### PR TITLE
add get mission hub path and redirect users landing at mhub.cc at a n…

### PR DESCRIPTION
**Pull request by jesusfreak3**
this should work, I cant really test it, but it should work.

### CODE REVIEW

File: app/assets/javascripts/angular/missionhubApp.run.js [hunk 1]
```diff
@@ -39,6 +41,9 @@ angular
             if (transition.to().data && transition.to().data.isPublic)
                 return true;
 
+            if ($location.host() === 'mhub.cc')
```
Comment (OzzieOrca) [inline app/assets/javascripts/angular/missionhubApp.run.js]: Does this cover `stage.hub.cc` also?

Comment (jesusfreak3) [reply app/assets/javascripts/angular/missionhubApp.run.js]: no, right now it only handles mhub.cc, if you want me to add stage.hub.cc as well, then I can add env variables for this, to catch it based on env let mek now

Comment (OzzieOrca) [reply app/assets/javascripts/angular/missionhubApp.run.js]: Yes I want it added. You could just check if the hostname contains `mhub.cc` here or check with a regex that it ends in that.

Comment (jesusfreak3) [reply app/assets/javascripts/angular/missionhubApp.run.js]: @OzzieOrca  ok just pushed i just used a simple includes on the string.

**Pull request merged**